src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,22 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
```

Comment:
Perhaps a class should have a header comment to inform users about what the class does. Same suggestion for the other .java files